plugin.yaml

```diff
@@ -2,4 +2,9 @@ name: "purser"
 shortDesc: "Cost Insight integration with kubernetes"
 longDesc: >
   Purser gives cost insights of kubernetes deployments.
-command: purser_plugin $@
+command: "./purser"
```

Comment:
let it be `purser_plugin $@`, if we change this we need to change all installation steps too. For now lets keep the command same.